nina_xmpp/__init__.py

```diff
@@ -295,4 +299,7 @@ def feeds(self, jid, arg):
                 url=url,
                 last_modified=last_modified,
             ))
-        return '\n'.join(feeds)
+        return '\n'.join(feeds + [
+            '',
+            _('The feeds are checked for updates every {} seconds').format(self.config['check_interval']),
```

Comment:
It would be nice to format the interval properly into minutes (if applicable). But that's totally optional.

Reply:
> Please also update `nina_xmpp/locale/nina_xmpp.pot` with the new translation strings.
> 
> Since obtaining the version might prove difficult (see line comment), it might make sense to split off the interval info into its own PR.

Updated, sorry for the delay

Reply:
If your fine with it, I would leave that out for now. Mostly because I'm too lazy to find a nice format and deal with things like minute/s and second/s (in the translations..)